### PR TITLE
1-1_show list of sport locations

### PR DESCRIPTION
**Pull request by marvinRoellecke**
- create new project with create next-app (typescript)
- connect project with github
- deploy app on vercel
- create folder structure (utils, lib, img, fonts)
- create dummyArray "dummyItems"
- create components [Header, Main, ItemListGeneral, Item]
- render ul with .map in ItemListGeneral
- set global styles in GlobalStyles.js
- used styledComponents for styling

### CODE REVIEW

File: components/GlobalStyles.js [hunk 1]
```diff
@@ -3,19 +3,34 @@ import { createGlobalStyle } from "styled-components";
...
---color-accent: #ABABAB;
+        --color-background: #FFFFFF;
+        --color-foreground: #000000;
+        --color-accent: #ABABAB;
+        --box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
     }
+
+    * {
+        box-sizing: border-box;
+        xborder: 1px solid black
```
Comment (ufelgen) [inline components/GlobalStyles.js:14]: here is an obsolete border property

File: components/Main/Main.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+import ItemListGeneral from "../ItemListGeneral/ItemListGeneral";
+
 export default function Main() {
-  return <p>Main Part</p>;
+  return (
+    <main>
+      <ItemListGeneral />
+    </main>
```
Comment (ufelgen) [inline components/Main/Main.js:7]: I haven't cloned this repo, so at this point I cannot see where you call the component Main (in the index.js probably). Maybe you need it later for additional purposes but if the only reason for your Main component is to wrap something in main tags, then maybe you can just add those in the index.js? Maybe the coaches have an opinion on this :)

**Review by ufelgen — COMMENTED** [review]
looks good to me! I only made some minor comments

File: utils/data/dummyItems.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const dummyItems = [
```
Comment (julialoeschel) [inline utils/data/dummyItems.js:1]: please use good names, always ;)

File: components/GlobalStyles.js (hunk 1 above)
Comment (marvinRoellecke) [reply components/GlobalStyles.js:14]: Ulli, thanks for reviewing my code! :)
I'm using this border property on and off to get a better overview of the rendered html elements.
Therefore I would like to keep this property.

File: components/Main/Main.js (hunk 2 above)
Comment (marvinRoellecke) [reply components/Main/Main.js:7]: correct, the "Main" component is called in pages/index.js.
Hmm..I think both ways work. For me it is more clear as written above. I will ask what's best practice.

File: utils/data/dummyItems.js (hunk 3 above)
Comment (marvinRoellecke) [reply utils/data/dummyItems.js:1]: alright ;)

**Pull request merged**